pkg/plugin/registry/organization.go

```diff
@@ -18,12 +20,36 @@ func ParseProviderName(name string) (string, string, error) {
 		return strings.ToLower(names[0]), names[1], nil
 	}
 	if len(names) == 1 {
-		return defaultOrganization, name, nil
+		return DefaultOrganization, name, nil
 	}
-	return "", "", fmt.Errorf("invalid provider name %s", name)
+	return "", "", fmt.Errorf("invalid provider name %q", name)
 }
 
 // ProviderRepoName returns a repository name for a given provider name.
 func ProviderRepoName(name string) string {
 	return fmt.Sprintf("cq-provider-%s", name)
 }
+
+// ParseProviderNameWithVersion parses the given string as "repository/provider@version" or "provider@version" (or @version omitted, defaulting to "latest")
+func ParseProviderNameWithVersion(nameWithVersion string) (string, string, string, error) {
+	versionParts := strings.Split(nameWithVersion, "@")
```

Comment:
I think we should be documenting what characters can be in a provider name

Reply:
<img width="318" alt="Screen Shot 2022-02-23 at 17 35 15 " src="https://user-images.githubusercontent.com/223029/155374993-3e65e096-cb11-49ea-9e9f-f0206713aeec.png">

Yes but a repo with `@` in the name is not possible with github anyway...

Reply:
For my own knowledge, do providers have to be on Github? I know that policies can be in many different locations

Reply:
Looks like it can be anywhere as long as it's indexed in our Firestore (`registry/hub.go`)